q2_fondue/entrezpy_clients/_pipelines.py

```diff
@@ -30,11 +30,33 @@ def _get_run_ids_from_projects(email, n_jobs, project_ids, log_level) -> list:
         {'db': 'sra'},
         analyzer=ELinkAnalyzer(), dependency=es
     )
-    # given SRA run IDs, fetch all metadata
+    # given BioProject IDs, fetch all metadata (incl. run IDs)
```

Comment:
This line should stay as-was. This step fetches metadata given run IDs received from elink step above, not BioProject IDs.